src/components/SearchPage.tsx

```diff
@@ -82,39 +69,39 @@ const SearchPage = () => {
             value={formData.pokemonName}
             ref={inputRef}
           />
-          {!isLoaded ? <div className="loader"></div> : null}
+          {isLoading && <div className="loader"></div>}
```

Comment:
I would always do it the other way:

```suggestion
          {!isLoaded ? <div className="loader"></div> : null}
```

Check out [this article](https://kentcdodds.com/blog/use-ternaries-rather-than-and-and-in-jsx).